netweaver/monitoring.sls

```diff
@@ -14,8 +14,13 @@ sap_host_exporter_exporter_service:
       - pkg: prometheus_sap_host_exporter_pkg
       - file: sap_host_exporter_configuration
 
-# for each node the instance number changes so we need to adapt configuration
+# the instance number of a node is unique and changes so we need to adapt configuration
 {% for node in netweaver.nodes if host == node.host %}
+
+{% set instance = node.instance %}
```

Comment:
Just to the same we do to get the instance in every place else:
```{% set instance = '{:0>2}'.format(node.instance) %}```

Reply:
```'{:0>2}'``` what is doing this by curiosity?